components/youcanbook_me/sources/common/booking-base.mjs

```diff
@@ -0,0 +1,28 @@
+import youcanbook_me from "../../youcanbook_me.app.mjs";
+
+export default {
+  props: {
+    youcanbook_me,
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 15 * 60,
+      },
+    },
+  },
+  methods: {
+    async emitEvent() { },
```

Comment:
Shouldn't we throw an Exception to avoid forgetting to implement this method in each trigger?